app/components/message_threads_bulk_actions_component.html.erb

```diff
@@ -98,15 +98,35 @@
               </div>
             <% end %>
 
-            <div class="py-1" role="none">
+            <%= content_tag(:div,
+              {
+                "data-controller": "message-drafts",
+                "data-message-drafts-thread-ids": @ids.to_s,
+                "data-message-drafts-submit-button-id": "confirmed_submit_button",
+                "data-message-drafts-submit-form-id": "bulk_actions_form",
+                "data-message-drafts-authenticity-token": form_authenticity_token,
+                "class": "py-1",
+                "role": "none"
+              }
+            ) do %>
+              <%= button_tag type: "button",
+                             "data-action": "click->message-drafts#bulkSubmit",
+                              class: "w-full text-gray-700 group flex items-center px-4 py-2 text-sm hover:bg-gray-100 hover:text-gray-900",
+                              data: { turbo_frame: "_top" },
+                              role: "menuitem" do %>
+                <%= render Common::IconComponent.new("paper-airplane", classes: "mr-2 sm:mr-3 h-5 w-5 text-gray-400 group-hover:text-gray-500") %>
+                Odoslať
```

Comment:
Zvazujem ci celu tuto logiku proste nespravit tak, ze pokial tam je nieco nepodpisane, tak to proste hodi error a nedovoli to poslat. Mame nejaky dovod posielat nepodpisane? Ved stale mozem dat prec ten request nie?

Tato JS logika mi pride velmi custom a nie uplne hotwire/stimulus styl. stimulus controlers su vacsinou take, ze sa daju pouzivat napriec apkami.

Reply:
Nakolko by sa zmenil flow, keby namiesto alertu nedovolime taketo spravy odoslat? Napr. pri hromadnom odosielani vyberies vlakna, ktore obsahuju drafty bez vyziadaneho podpisu. Mozes kliknut odoslat, odpali sa to a potom dostanes flash message, ze to nepreslo, lebo chybaju podpisy? Alebo inak by to vyzeralo?

Reply:
Dalo by sa aj tak, bolo by to implementacne asi cistejsie.

Reply:
@luciajanikova podla mna to dajme takto jednoducho lebo dost si tam tahame komplexitu, ktora je vlastne nejaky okrajovy stav.